includes/tasks/class-wc-calypso-task-setup-woocommerce-square.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Task;
+
+/**
+ * WCBridgeSetupWooCommerceSquare Task
+ *
+ * @since   2.0.11
+ * @version 2.1.10
+ */
+class WCBridgeSetupWooCommerceSquare extends Task {
+	/**
+	 * ID.
+	 *
+	 * @return string
+	 */
+	public function get_id() {
+		return 'setup-woocommerce-square';
+	}
+
+	/**
+	 * Title.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		return __( 'Get paid with Square', 'woocommerce' );
```

Comment:
Since this string is not in core, we'll need to make the translations in bridge. Also please apply to all in this file

```suggestion
		return __( 'Get paid with Square', 'wc-calypso-bridge' );
```